### ISSUE TITLE
FHIR RDF ontology URLs do not resolve

### ISSUE THREAD

**Issue by dbooth-boston** [w3c/hcls-fhir-rdf#48]
https://www.w3.org/2017/03/28-hcls-minutes.html#item02

For example, at the bottom of http://hl7.org/fhir/diagnosticreport-example-f001-bloodexam.ttl.html

```
# - ontology header ------------------------------------------------------------

<http://hl7.org/fhir/DiagnosticReport/f001.ttl> a owl:Ontology;
  owl:imports fhir:fhir.ttl;
  owl:versionIRI <http://hl7.org/fhir/STU3/DiagnosticReport/f001.ttl> .
```
both http://hl7.org/fhir/DiagnosticReport/f001.ttl and the versioned URI http://hl7.org/fhir/STU3/DiagnosticReport/f001.ttl return 404 when dereferenced.

ACTION: Harold to initiate discussion with Grahame about adding conneg and a variable for ont version.

**Comment by hsolbrig** [comment]
Resolved on Grahame's test server and http://hl7.org.   
Example:
   wget --header "Accept:text/turtle" http://hl7.org/fhir/Patient/f001
returns the .ttl record

Note, however, that:
wget --header "Accept:text/turtle" http://hl7.org/fhir/Patient

Still does not work.

**Comment by dbooth-boston** [comment]
More discussion; https://www.w3.org/2017/10/24-hcls-minutes.html#item05

**Comment by dbooth-boston** [comment]
Media type and file extension for OWL functional notation (Manchester syntax): https://www.w3.org/TR/owl2-manchester-syntax/#Appendix:_Internet_Media_Type.2C_File_Extension_and_Macintosh_File_Type

**Comment by dbooth-boston** [comment]
This issue should also cover version URIs in the ontology header: https://www.w3.org/2017/10/24-hcls-minutes.html#item07

**Comment by dbooth-boston** [comment]
@balhoff Do you know if the ontology URLs resolve now?

**Comment by balhoff** [comment]
@dbooth-boston yes, the ontology downloaded from https://www.hl7.org/fhir/downloads.html has ontology IRI `http://hl7.org/fhir/fhir.ttl`, and that ontology can be loaded from that URL. It imports an ontology `http://hl7.org/fhir/w5.ttl`, and that one correctly resolves as well.

**Comment by balhoff** [comment]
The ontology version IRI `http://hl7.org/fhir/R5/fhir.ttl` also seems to resolve correctly. The w5 ontology has a broken version IRI. It is stored as a string rather than an IRI:

```turtle
fhir:w5.ttl owl:versionIRI "http://hl7.org/fhir/R5/w5.ttl" .
```

That should be changed to an IRI (but it does resolve).

**Comment by dbooth-boston** [comment]
Discussed on today's call: https://www.w3.org/2025/05/01-hcls-minutes.html#2d2b

**Comment by dbooth-boston** [comment]
Closing, as this appears to have been fixed.
